src/Riok.Mapperly/Riok.Mapperly.csproj

```diff
@@ -4,11 +4,12 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageProjectUrl>https://riok.github.io/mapperly</PackageProjectUrl>
     <PackageId>Riok.Mapperly</PackageId>
     <PackageLicense>Apache-2.0</PackageLicense>
     <PackageDescription>
       A .NET source generator for generating object mappings.
-      Trimming save.
+      Without any reflection.

```

Comment:
On a second thought, this sounds a little weird... Maybe change it to "Does not use reflection"? Or something else